src/main/java/org/opentripplanner/middleware/otp/OtpDispatcherImpl.java

```diff
@@ -42,6 +51,11 @@ private URI buildUri(String params, String endPoint) {
         return uri;
     }
 
+    /**
+     * Makes a call to the OTP server plan end point and converts, if possible, the response into POJOs. The POJOs,
+     * original response and status are wrapped in a single object and returned. It will fail if a connection is not
+     * made or is unable to parse the plan response.
+     */
```

Comment:
I'm just not sure if we want to really fail to respond just because we couldn't parse the response. To me, it's acceptable if failure to parse ends up meaning we cannot store some results, but I failing to respond to the client on parse failure seems like something we want to avoid.

Reply:
In other words, it seems like we should move the parsing logic outside of this dispatcher and into the OtpRequestProcessor.

Reply:
Have implemented this.